backend/docker/docker-compose-full.yml

```diff
@@ -40,6 +40,12 @@ services:
             - "8080:8080"
             - "8081:8081"
 
+    yorkie-intelligence:
+        image: "ollama/ollama:latest"
+        restart: always
+        ports:
+            - ${OLLAMA_HOST_PORT}:11434
```

Comment:
Already mentioned.

And I have a question.
Do we need to expose the Ollama container port to the user?